src/templates/index.html

```diff
@@ -44,12 +44,22 @@ <h4 class="card-title">CubingUSA Nationals</h4>
       </div>
       <div class="standard-block">
         <div class="card">
-          <img class="card-img-top" src="/static/img/cubingUSA_4.svg" alt="Regional Championships">
+          <!-- <img class="card-img-top" src="/static/img/cubingUSA_4.svg" alt="Regional Championships">
           <div class="card-body">
             <h4 class="card-title">Regional Championships</h4>
             <p class="card-text">Each year, we host seven Regional Championships across the country, allowing lots of different cubers a chance to compete for a title.</p>
             <a href="{{ c.uri_for('competitions_regional') }}" class="button">Learn More</a>
-          </div>
+          </div> -->
+          <!-- <div class="card-header"><h4 class="card-title">Recent Posts</h4></div> -->
+
+          <ul class="list-group list-group-flush">
+            <li class="list-group-item active"><h3>Recent Posts</h3></li>
+            <li class="list-group-item">Test Posts <a href="cubingusa.org">Read More</a></li>
+            <li class="list-group-item">Will Appear Here <a href="cubingusa.org">Read More</a></li>
+            <li class="list-group-item">Regional Championships Announced! <a href="cubingusa.org">Read More</a></li>
+            <li class="list-group-item">Make sure to check out our new logo!</li>
+          </ul>
+
```

Comment:
How does this look? I have it set up so it's different from the 'more links' card because it seems more important. I have the layout for making it a normal card header commented out for now. Do people want this to look different, or have it the same as the 'more links' style?